### PR TITLE
Starter

### PR DESCRIPTION
**Pull request by KoxyG**

### CODE REVIEW

**Comment by KoxyG** [comment]
there is a need to merge old changes inorder for me to be able to push new changes.

File: client/.env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+VITE_PUBLIC_ETH_CONTRACT_ADDRESS=0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7
+VITE_PUBLIC_ACCOUNT_CLASS_HASH=0x04d07e40e93398ed3c76981e72dd1fd22557a78ce36c0515f679e27f0bb5bc5f
+VITE_PUBLIC_MASTER_ADDRESS=0x517ececd29116499f4a1b64b094da79ba08dfd54a3edaa316134c41f8160973
+VITE_PUBLIC_MASTER_PRIVATE_KEY=0x1800000000300000180000000000030000000000003006001800006600
+VITE_PUBLIC_WORLD_ADDRESS=0x223b959926c92e10a5de78a76871fa40cefafbdce789137843df7c7b30e3e0
+VITE_PUBLIC_NODE_URL=http://localhost:5050
+VITE_PUBLIC_TORII=http://localhost:8080
```
Comment (milancermak) [inline client/.env]: Please put `.env` file to `.gitignore`.

In general, *never* commit private keys or other data that can be misused.

File: client/.gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "dojo-starter"]
```
Comment (milancermak) [inline client/.gitmodules:1]: We should not need this (or any) submodule for the dojo-duel project.

**Review by milancermak — CHANGES_REQUESTED** [review]

**Review by KoxyG — COMMENTED** [review]
i just and added .env to gitignore, pls check

**Pull request closed without merging**